__tests__/main.test.ts

```diff
@@ -162,3 +163,32 @@ test('processing a milestone with only a few issues will not close it', async ()
 
   expect(processor.closedMilestones.length).toEqual(0);
 });
+
+test('processing a closed milestone detecting new open issues related, will reopen it', async () => {
```

Comment:
Can we add a test to make sure processing a closed milestone with closed issues doesn't reopen it?

Reply:
yes adding